PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,13 +5,14 @@
 
 ---
 
+
 ## Changes
 - add
 - remove
 - modify
 
-## Definition of Done (from ticket)
+- #TICKET_NUMBER
 
 ## QA steps (optional)
 
-## Known Issues
+## Known Issues or Open Questions
```

Comment:
I usually place comments directed at the reviewer in the PR itself, and rarely use this. But if others use it, I'd definitely keep it.  I suppose this is similar to a section we used on Meta team called "Notes for Reviewer" (or something similar).

Reply:
One question... how will known issues/open questions noted in a PR be tracked or addressed in the future?

Reply:
I like the section for more meta comments/questions. Like a question about the validity of an overall approach. I like "Notes for Reviewer" I'll swap that in.